pkg/operator/hive/hive.go

```diff
@@ -31,14 +33,17 @@ import (
 	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
 
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
 const (
-	dnsServersEnvVar = "ZONE_CHECK_DNS_SERVERS"
+	dnsServersEnvVar       = "ZONE_CHECK_DNS_SERVERS"
+	hiveAdditionalCASecret = "hive-additional-ca"
```

Comment:
Probably worth godoc that this secret contains all the HiveConfig additional CAs assembled into one.